android/src/main/java/com/reactnativekeyboardcontroller/KeyboardControllerViewManager.kt

```diff
@@ -32,6 +32,17 @@ class KeyboardControllerViewManager(reactContext: ReactApplicationContext) : Rea
     val window = activity.window
     val decorView = window.decorView
 
+    val callback = TranslateDeferringInsetsAnimationCallback(
+      view = view,
+      persistentInsetTypes = WindowInsetsCompat.Type.systemBars(),
+      deferredInsetTypes = WindowInsetsCompat.Type.ime(),
+      // We explicitly allow dispatch to continue down to binding.messageHolder's
+      // child views, so that step 2.5 below receives the call
+      dispatchMode = WindowInsetsAnimationCompat.Callback.DISPATCH_MODE_CONTINUE_ON_SUBTREE,
+      context = mReactContext
+    )
+    ViewCompat.setWindowInsetsAnimationCallback(decorView, callback)
+    ViewCompat.setOnApplyWindowInsetsListener(decorView, callback)
```

Comment:
Many libraries may rely on `decorView` and setup their own handlers (for example `react-native-screens` when `translucent` status bar prop is set). So the handler may be overwritten

Maybe we can setup listeners on our view?